src/test/run-pass/optional_coma_in_match_arm.rs

```diff
@@ -0,0 +1,35 @@
+fn main() {
```

Comment:
Aha! That's why I am always not sure if it's a coma or a comma: they spelled differently : ) ! Thanks!